tools/reactive-controllers/src/FocusGroup.ts

```diff
@@ -344,6 +368,7 @@ export class FocusGroupController<T extends HTMLElement>
 
     addEventListeners(): void {
         this.host.addEventListener('focusin', this.handleFocusin);
+        this.host.addEventListener('click', this.handleClick);
```

Comment:
should we clean up this event listener below?

Reply:
Yes!!